### ISSUE TITLE
Add an option to always use async/await

### ISSUE THREAD

**Issue by maca88** [maca88/AsyncGenerator#97]
Instead of:
```
public Task ReadAsync() {
  return File.ReadAsync();
}
```
would be:
```
public async Task ReadAsync() {
  await File.ReadAsync();
}
```

**Comment by hazzik** [comment]
Thanks

**Comment by EamonHetherton** [comment]
Will this be on or off by default as it does have some performance overhead as I understand it.

**Comment by maca88** [comment]
> Will this be on or off by default as it does have some performance overhead as I understand it.

This option will be off by default

**Comment by hazzik** [comment]
@maca88 can this be released? Either in 0.8.2.x branch or in master

**Comment by maca88** [comment]
Released in `0.12.0` and `0.8.2.6`.
